stdpopsim/dfe.py

```diff
@@ -31,28 +37,97 @@ class MutationType(object):
     exponential and gamma, a negative mean can be provided, obtaining always
     negative values.
 
+    Instead of a single dominance coefficient, a discretized relationship
+    between dominance and selection coefficient can be implemented:
```

Comment:
```suggestion
    Instead of a single dominance coefficient (variable 'dominance_coeff'), a 
    discretized relationship between dominance and selection can be 
    implemented:
```